acceptance_tests/features/pages/inbox_internal.py

```diff
@@ -40,6 +44,10 @@ def get_no_messages_text():
     return browser.find_by_text('No new conversations')
 
 
+def get_no_closed_conversations_text():
```

Comment:
Some of these methods are the same in both, wondering if they should go in a common type file? I'll leave it to you as they arent massive methods.